src/components/Table/components/cell.tsx

```diff
@@ -70,12 +65,7 @@ export const headerRenderFn: (column: GridColumnWithId<any>, as: RenderAs, colSp
   (column, as, colSpan) => (key, css, content, row, rowStyle, classNames: string | undefined, onClick, tooltip) => {
     const Cell = as === "table" ? "th" : "div";
     return (
-      <Cell
-        key={key}
-        css={{ ...css, ...tableRowStyles(as, column) }}
-        className={classNames}
-        {...(as === "table" && { colSpan })}
-      >
+      <Cell key={key} css={{ ...css }} className={classNames} {...(as === "table" && { colSpan })}>
```

Comment:
For whatever reason this could become `css={css}` due to its type being `Properties`. It expected something else, but I didn't feel like digging into it too deep. I'm hoping we'll update these at somepoint to be their own components, so I didn't want to spend too much time on it.